components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/cache/SessionDataCacheEntry.java

```diff
@@ -136,4 +137,24 @@ public void setSessionContextIdentifier(String sessionContextIdentifier) {
 
         this.sessionContextIdentifier = sessionContextIdentifier;
     }
+
+    /**
+     * Get removeOnConsume.
+     *
+     * @return removeOnConsume.
+     */
+    public boolean isRemoveOnConsume() {
+
+        return removeOnConsume;
+    }
+
+    /**
+     * Set removeOnConsume.
```

Comment:
Can we improve the comment to explain the purpose of this attribute

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2072/commits/042971f0912e5080e6171244828d625f5bcacb80